server/zanata-war/src/main/webapp/resources/zanata/multi-file-upload.xhtml

```diff
@@ -216,7 +219,7 @@ site: http://www.fsf.org.
               <div id="#{cc.clientId}-fileupload-advanced"
                 class="l--pad-all-half bg--pop-higher is-hidden">
                 <label for="#filepath">#{msgs['jsf.upload.FilePath']}</label>
-                <input type="text" id="filepath" name="filepath"/>
+                <input type="text" id="filepath" maxlength="250" name="filepath"/>
```

Comment:
@djansen-redhat Why is this one only 250?

Reply:
250 + filename of 4 characters (oops :laughing: )